pkg/vpcmodel/groupingSelfLoop.go

```diff
@@ -87,12 +87,8 @@ func (g *GroupConnLines) relevantKeysToCompare(groupingSrcOrDst map[string][]*gr
 		if lines[0].isSrcOrDstExternalNodes() {
 			continue
 		}
-		// if vsi then the subnets must be equal; if not vsis then empty string equals empty string
-		if g.getSubnetUIDIfVsi(lines[0].src) != g.getSubnetUIDIfVsi(lines[0].dst) {
-			continue
-		}
-		// if subnets then the vpc must be equal; if not subnets then empty string equals empty string
-		if g.getVPCUIDIfSubnet(lines[0].src) != g.getVPCUIDIfSubnet(lines[0].dst) {
+		// if vsi then the subnets of src and dst must be equal; similarly if subnet then vpcs musy be equal
+		if g.getUIDSubnetOrVPC(lines[0].src) != g.getUIDSubnetOrVPC(lines[0].dst) {
```

Comment:
how are these changes related to this PR ? (they were done on the other open PR, not related here)

Reply:
I did the changes you requested in the other PR here by mistake and removed them. Seems I did not push.